invenio_records_resources/services/records/service.py

```diff
@@ -579,3 +581,73 @@ def on_relation_update(
 
             self.reindex(identity, search_query=search_query)
         return True
+
+    @unit_of_work()
+    def create_or_update_many(self, identity, data, uow=None):
+        """Create or update a list of records.
+
+        This method takes a list of record data and creates or updates the corresponding records.
+
+        :param identity: The user identity performing the operation.
+        :param data: A list of tuples containing the record ID and record data.
+        :param uow: The unit of work to register the record operations. Defaults to None.
+
+        Returns:
+            list: A list of tuples containing the operation type ('create' or 'update'), the processed record or the record dict, and any schema errors encountered.
```

Comment:
nit: the format of the docstring here suggest we use typing (list:) which is not the case

Reply:
Unless I am missing something orn ot understanding the comment, we are in fact returning a list of tuples

Reply:
I am suggesting to remove the name of the returned type, not sure if we have it in any other docstring